Makefile

```diff
@@ -295,7 +295,12 @@ goimports:
 		MODIFIED_FILES=$$(git diff --name-status $$MERGE_BASE -- | cut -f2 | grep '.go$$' || true) && \
 		echo "Merge base: $$MERGE_BASE" && \
 		echo "Modified files: $$MODIFIED_FILES" && \
-		if [ -n "$$MODIFIED_FILES" ]; then $(GOIMPORTS) -w $$MODIFIED_FILES; fi
+		if [ -n "$$MODIFIED_FILES" ]; then $(GOIMPORTS) -local "go.temporal.io" -w $$MODIFIED_FILES; fi
+
+goimports-all:
+	@printf $(COLOR) "Run goimports for modified files..."
+	@UNGENERATED_FILES=$$(find . -type f -print0 | xargs -0 grep -L -e "Code generated by .* DO NOT EDIT." | grep '.go$$' || true) && \
```

Comment:
```suggestion
	@UNGENERATED_FILES=$$(find . -type f -name '*.go' -print0 | xargs -0 grep -L -e "Code generated by .* DO NOT EDIT." || true) && \
```

Reply:
Excellent, forgot this was a grep option.